app/src/main/java/com/group80/uoftinder/entities/User.java

```diff
@@ -0,0 +1,129 @@
+package com.group80.uoftinder.entities;
+
+import android.net.Uri;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+
+public class User {
```

Comment:
I would vouch for more documentation throughout, but I think at this stage before Milestone 4, at least User class should have some simple javadocs! It's a lot of simple work to add javadoc for all the getters and setters in here so hopefully not too bad :)

Reply:
I will do this in the User branch (0c) so everyone can have the javadocs when merging with that branch!